integration-cli/docker_cli_start_volume_driver_unix_test.go

```diff
@@ -318,33 +291,29 @@ func (s *DockerExternalVolumeSuite) TestStartExternalVolumeDriverRetryNotImmedia
 
 	errchan := make(chan error)
 	go func() {
-		if out, err := s.d.Cmd("run", "--rm", "--name", "test-data-retry", "-v", "external-volume-test:/tmp/external-volume-test", "--volume-driver", "test-external-volume-driver-retry", "busybox:latest"); err != nil {
-			errchan <- fmt.Errorf("%v:\n%s", err, out)
-		}
+		out, err := s.d.Cmd("run", "--rm", "--name", "test-data-retry", "-v", "external-volume-test:/tmp/external-volume-test", "--volume-driver", "test-external-volume-driver-retry", "busybox:latest")
+		c.Assert(err, checker.IsNil, check.Commentf(out))
```

Comment:
this won't fail the test if an error arises since it's in a goroutine, this change should be reverted